src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,32 @@
+
+.catalog {
+  --grid-column-count: 1;
+
+  display: grid;
+
+  padding: 50px 40px;
+  padding-bottom: 0;
+
+  grid-template-columns: repeat(var(--grid-column-count), 200px);
```

Comment:
move card width to a variable since you've used this value here and at `product-card.scss` as well